automation/jenkins_build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Set env var
+ACCESS_KEY=$1
```

Comment:
Sure, by exporting them as environment variables. Will still have to export them in jenkins configuration shell script